doc/common/openstack-release.md

```diff
@@ -1,32 +1,32 @@
-On the occurrence of any of the triggering events, the TSC shall constitute an OpenStack Release Selection Committee composed of a maximum of 7 (seven) members representing both operators and vendors. These committee members shall be from active CNTT member organisations and meet the criteria specified for “[Voting Representatives]( https://github.com/cntt-n/CNTT/blob/master/doc/gov/chapters/chapter05.md#52-voting-representatives)”. 
+On the occurrence of any of the triggering events, the TSC shall constitute an OpenStack Release Selection Committee composed of a maximum of 7 (seven) members representing both operators and vendors. These committee members shall be from active Anuket member organisations and meet the criteria specified for “[Voting Representatives](../gov/chapters/chapter05.md#52-voting-representatives)”.
 The committee decisions shall be by consensus and no decision shall be made without at least 5 members agreeing.
 The committee may agree by unanimous agreement to adjust the OpenStack Release Selection Criteria.
 
 ## OpenStack Release Selection Criteria
-The OpenStack Release Selection Committee shall utilize the following criteria, and any other criteria that it unanimously agrees to, in selecting the next CNTT OpenStack baseline release:
+The OpenStack Release Selection Committee shall utilize the following criteria, and any other criteria that it unanimously agrees to, in selecting the next Anuket OpenStack baseline release:
 -	The latest OpenStack release that was released approximately 6 months ago
     - The Committee may agree to relax or extend the 6 months period based on its knowledge of OpenStack releases
 -	The OpenStack release should be supported by the OPNFV Installer (Airship)
 -	Backward Compatibility: ensure API support
 -	Consider OpenStack Distribution vendors and their extended support versions
 
-## Deprecation of CNTT OpenStack Releases
-CNTT shall support no more than 2 (two) OpenStack releases at any given time. Thus, on selection of a new CNTT OpenStack baseline release, an existing CNTT OpenStack release may be deprecated. The selection of new CNTT OpenStack release n, triggers the deprecation of the n-2 release. On the completion of the Reference Architecture for release n, the release n-2 will stand deprecated.
-Please note that reference to releases in this subsection is to CNTT’s OpenStack release where Pike is release 1. 
+## Deprecation of Anuket OpenStack Releases
+Anuket shall support no more than 2 (two) OpenStack releases at any given time. Thus, on selection of a new OpenStack baseline release of the Anuket specifications, an existing Anuket OpenStack release may be deprecated. The selection of new release n triggers the deprecation of the n-2 release. On the completion of the Reference Architecture for release n, the release n-2 will stand deprecated.
+Please note that reference to releases in this subsection is to Anuket’s OpenStack release where Train is release 1.
```

Comment:
Pike is release 1, Train is release 2
Please do not change content as part of replacing CNTT with Anuket